aes/decryptor.go

```diff
@@ -0,0 +1,122 @@
+package aes
```

Comment:
I agree, I thought about this too, but for some reason I veered off and started working on something else.